docs/rfds/acp-agent-registry.mdx

```diff
@@ -0,0 +1,125 @@
+| `capabilities`   | Array of ACP method names implemented (e.g. `["terminal/new","files/read"]`).                                                                                                                                                                                                                                                                                      |
+| `auth`           | Array of auth options for authentication. This is the trickiest part of the schema.                                                                                                                                                                                                                                                                                |
+| `distribution`   | _Optional._ Object mapping target platforms to download/execution info. Each target key follows `<os>-<arch>` format (e.g., `darwin-aarch64`, `linux-x86_64`, `windows-x86_64`). Each target specifies `archive` (download URL), `cmd` (executable path), optional `args` (array of command-line arguments), and optional `env` (object of environment variables). |
+
+Example skeleton:
+
+```jsonc
+{
+  "id": "someagent",
+  "name": "SomeAgent",
+  "version": "1.0.0",
+  "schema_version": "1",
+  "description": "Agent for code editing",
+  "homepage": "https://github.com/example/someagent",
+  "repository": "https://github.com/example/someagent",
+  "authors": ["Example Team"],
+  "license": "MIT",
+  "capabilities": ["terminal", "fs/read", "fs/write"],
+  "auth": [
+    {
+      "type": "api_key",
+    },
+  ],
+  "distribution": {
+    "darwin-aarch64": {
+      "archive": "https://github.com/example/someagent/releases/latest/download/someagent-darwin-arm64.zip",
+      "cmd": "./someagent",
+      "args": ["acp"],
+    },
+    "darwin-x86_64": {
+      "archive": "https://github.com/example/someagent/releases/latest/download/someagent-darwin-x64.zip",
+      "cmd": "./someagent",
+      "args": ["acp"],
+    },
+    "linux-aarch64": {
+      "archive": "https://github.com/example/someagent/releases/latest/download/someagent-linux-arm64.zip",
+      "cmd": "./someagent",
+      "args": ["acp"],
+    },
+    "linux-x86_64": {
+      "archive": "https://github.com/example/someagent/releases/latest/download/someagent-linux-x64.zip",
+      "cmd": "./someagent",
+      "args": ["acp"],
+    },
+    "windows-x86_64": {
+      "archive": "https://github.com/example/someagent/releases/latest/download/someagent-windows-x64.zip",
+      "cmd": "./someagent.exe",
+      "args": ["acp"],
+      "env": {
+        "SOMEAGENT_MODE_KEY": "",
+      },
```

Comment:
Since this is a spec, should we care about knowing if the secret is required or not?